policyengine_canada/parameters/gov/provinces/nt/tax/income/benefits/child_benefit/younger/age_eligibility.yaml

```diff
@@ -0,0 +1,11 @@
+description: Northwest Territoriesa provides an enhanced benefit amount for children below this age under the child benefit.
```

Comment:
description: Northwest Territories provides an enhanced child benefit amount to filers below this age threshold.